templates/strings/objc-h.stencil

```diff
@@ -0,0 +1,50 @@
+// Generated using SwiftGen, by O.Halligon — https://github.com/SwiftGen/SwiftGen
+
+{% if tables.count > 0 %}
+@import Foundation;
+
+NS_ASSUME_NONNULL_BEGIN
+
+{% macro parametersBlock types %}{% filter removeNewlines:"leading" %}
+  {% for type in types %}
+    ({% call paramTranslate type %})p{{ forloop.counter }}{% if not forloop.last %} and:{% endif %}
+  {% endfor %}
+{% endfilter %}{% endmacro %}
+{% macro argumentsBlock types %}{% filter removeNewlines:"leading" %}
+  {% for type in types %}
+    p{{forloop.counter}}{% if not forloop.last %}, {% endif %}
+  {% endfor %}
+{% endfilter %}{% endmacro %}
+{% macro paramTranslate swiftType %}
+  {% if swiftType == "Int" %}
+    NSInteger
+  {% elif swiftType == "String" %}
+    NSString*
+  {% else %} 
```

Comment:
No case for `Float` (or `Double`, I don't remember which one we use)?

Reply:
I found the list of types in `StringParser.swift`, `public enum PlaceholderType`, they are String (%@), Float (%[a,e,f,g]), Int (%[d,i,o,u,x]), Character (%c), UnsafePointer<unichar> (%s), UnsafePointer<Void> (%p), and UnsafePointer<()> (not used)".

BTW: %@ is mapped to NSString, the feedback I've gotten is that `id` is more correct.

Reply:
I'm sorely tempted to extend the code in StringParser.swift, creating a `typesObjC` attribute that would be an array of `ObjCPlaceholderType`.  But for now, I think I'll just extend the `paramTranslate` macro in the `.stencil` files.

Reply:
I added a case for `Float`.

I changed the output for all the things I don't handle to `FIX_OBJC_STRING_DOT_STENCIL_PLEASE`.  That should make it easier for the next guy to find where the fix goes.